infinispan-client/README.md

```diff
@@ -4,10 +4,16 @@ This example showcases how to use the Infinispan client with Quarkus.
 
 # Start the Infinispan server
 
-<!-- TODO: https://github.com/quarkusio/quarkus-quickstarts/issues/148
-- Running with Docker `docker run -it -p 11222:11222 jboss/infinispan-server:latest`
--->
-- Download the server (e.g. 9.4 or 10.0) from `http://www.infinispan.org/` and run `./bin/standalone.sh`
+You have two options:
+
+Option 1: Running with Docker `docker run -it -p 11222:11222 jboss/infinispan-server:latest`
+
+If you are using Docker For Mac, there is an known issue. Blog post [here](https://blog.infinispan.org/2018/03/accessing-infinispan-inside-docker-for.html)
```

Comment:
```suggestion
If you are using Docker For Mac, there is a known issue (see [this blog post](https://blog.infinispan.org/2018/03/accessing-infinispan-inside-docker-for.html for more details).
```

You probably will drop it if you include the `hotrod-client.properties` but who knows :).

Reply:
Ah so I see you included the file by default. Maybe we could drop the instructions then?

Reply:
+1 to drop the instructions here

Reply:
Maybe just reword? To explain that it is not mandatory if you are not using docker 4 mac

Reply:
I proposed a rewording. I prefer to explain why this file is there to avoid confusion around it. WDYT?